.github/workflows/e2e.yml

```diff
@@ -193,7 +198,7 @@ jobs:
         run: docker pull brightsec/cli:${{ inputs.version }}
 
       - name: Checkout Repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5 # v2
```

Comment:
Redundant changes; this is equivalent to using `actions/checkout@v2.7.0` or `actions/checkout@v2`. Please revert. Ditto elsewhere